src/components/breadcrumb/breadcrumb.js

```diff
@@ -15,16 +16,16 @@ class Breadcrumb extends React.Component {
   render() {
     const { t, threadName, threadId, totalTasks, taskPosition } = this.props;
     return (
-      <PfBreadcrumb className="integr8ly-breadcrumb pf-u-pl-lg">
-        <PfBreadcrumb.Item onClick={this.homeClicked}>
+      <PfBreadcrumb className="integr8ly-breadcrumb">
+        <BreadcrumbItem onClick={this.homeClicked} className="integr8ly-breadcrumb-home">
```

Comment:
You can delete this `className="integr8ly-breadcrumb-home"`, as we're doing a 'reset' to default PatternFly 4 styles on converted components.